src/pfe/portal/modules/Templates.js

```diff
@@ -109,6 +109,18 @@ module.exports = class Templates {
     return this.projectTemplates;
   }
 
+  async getTemplatesOfStyle(projectStyle) {
+    const allTemplates = await this.getTemplateList();
+    let relevantTemplates = allTemplates.filter(template =>
+      template.projectStyle === projectStyle
+    );
+    if (projectStyle === 'IBM Cloud') {
```

Comment:
Is this because 'IBM Cloud' projects don't always have a project style? The intention of this if statement is a bit unclear to me, the rest of the function looks nice

Reply:
Can you change 'IBM Cloud' to 'Codewind' please